apis/deployer/healthchecks/doc.go

```diff
@@ -0,0 +1,11 @@
+// SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors.
```

Comment:
think we should move them to `apis/deployer/utils/healthchecks` so that we do not mix them with other main deployer configurations

Reply:
Done